SCP/SCP.cpp

```diff
@@ -90,7 +90,7 @@ int main(int argument_count, char * argv[])
 
 	// TODO: Free all the memory we allocated
 	free_instance(&instance);
-	free(output_file_path);
+	//free(output_file_path);
```

Comment:
I get assertions and run-time errors when trying to execute this in my environment.  I don't know why, spent ages on it, but just wanted to get something running,  so I stopped looking into it and just commented it out so I could move on.  I can turn it back on for you - but until I know what's going on, I need it commented out for my environment.